src/applications/pensions/containers/ConfirmationPage.jsx

```diff
@@ -33,77 +27,124 @@ class ConfirmationPage extends React.Component {
 
     return (
       <div>
```

Comment:
I would recommend adding some space between the end of the content and the footer.
```suggestion
      <div className="vads-u-margin-bottom--9">
```